server/otlp/ingester.go

```diff
@@ -95,7 +140,26 @@ func (i ingester) Ingest(ctx context.Context, request *pb.ExportTraceServiceRequ
 	}, nil
 }
 
-func (i ingester) processTrace(ctx context.Context, modelTrace traces.Trace, ix int, requestType RequestType) error {
+func (i *ingester) increaseSpanCount(newSpansCount int) {
+	i.mutex.Lock()
+	defer i.mutex.Unlock()
+
+	i.spanCount += newSpansCount
```

Comment:
Sanity check question: Does this counter consider different counts for tenantIds?

Reply:
No, it doesn't. However, this ingester is only used by the core version, thus tenant ids are irrelevant. The agent ingester must care about it though.